### PR TITLE
Fix one link to /orgs/developers-team

### PR DESCRIPTION
**Pull request by ankon**

### CODE REVIEW

File: content/orgs/setup.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ added to the developers team.
 You may delete the developers team. If you do, newly added Organization
 members will not be added to any teams by default.
 
-For more information about the developers team, see [/orgs/developers-team]
+For more information about the developers team, see [developers team][8]
```
Comment (ashleygwilliams) [inline content/orgs/setup.md]: could u capitalize this? eg Developers Team

**Comment by ashleygwilliams** [comment]
thanks so much for this. just one not and then im happy to merge!

**Comment by ashleygwilliams** [comment]
err, nit*

**Comment by ankon** [comment]
@ashleygwilliams sure, no problem :)

**Comment by ashleygwilliams** [comment]
thanks for this!

**Pull request merged**